ql/ql/src/codeql_ql/style/docs/NonUSSpellingQuery.qll

```diff
@@ -18,7 +18,7 @@ predicate contains_non_us_spelling(string s, string wrong, string right) {
     wrong != "analyse"
     or
     // analyses (as a noun) is fine
-    s.regexpMatch(".*analyse[^s].*") and
+    s.regexpReplaceAll("[\\r\\n]", " ").regexpMatch(".*analyse[^s].*") and
```

Comment:
I wonder if this is faster or slower than using `[\\s\\S]` (IIRC)to match any character.

Reply:
I'm not sure, but I doubt it would be noticeable either way given the number of strings we're dealing with.

I'm about to gut this query to generalise it, and will likely replace this code, anyway.